src/App.js

```diff
@@ -1,32 +1,38 @@
-import React from 'react';
-import { Route, Switch } from 'react-router-dom';
-import Calculator from './components/calculator';
-import Navbar from './components/Navbar';
+/* eslint-disable */
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import Calculator from './components/Calculator';
```

Comment:
- [ ] Same here. The `c` should be small in `./components/**C**alculator'. Due to this, your application is not running and giving me the error.

Reply:
typo fix